### ISSUE TITLE
2019-11-20 tc011 and tc012 both trigger warnings concerning CONCOUTPUTUNIT not in proper form in unit_conversion routine

### ISSUE THREAD

**Issue by tensfeldt** [tensfeldt/openNCA#29]
Following occurs when executing tc012 but warnings from tc011 are similar:

```
> results_list <- run_computation(data=d, map=mct, flag=flags, parameterset=parameterset)
unit_conversion : Unit Class 1 (Time) time_col:  TMAX1 TMAX2 TMIN1 TMIN2 TLAST TLAG KELTMLO KELTMHI THALF LASTTIME MRTEVIFO1 MRTEVIFO2 MRTEVIFP1 MRTEVIFP2 TAU1 TAU2  parameters are scaled from  HR  to  HR  via scaling factor:  1 
unit_conversion : Unit Class 3 (Dose) dose_col:  DOSE1  parameters are scaled from  MG  to  MG  via scaling factor:  1 
unit_conversion : Unit Class 3 (Dose) dose_col:  DOSE2  parameters are scaled from  MG  to  MG  via scaling factor:  1 
unit_conversion : Unit Class 4 (Volume) volume_col:  VZFTAU1 VZFTAU2  parameters are scaled from  ML  to  L  via scaling factor:  0.001 
unit_conversion : Unit Class 5 (Amount/Volume) conc_col:  CMAX1 CMAX2 CMIN1 CMIN2 CLAST1 CLAST2 CTROUGH1 CTROUGH2 CTROUGHEND1 CTROUGHEND2 CAV1 CAV2  parameters are scaled from  UG/ML  to  NG/ML  via scaling factor:  1000 
unit_conversion : Unit Class 6 (1/Time) kel_col:  KEL  parameters are scaled from  1/HR  to  1/HR  via scaling factor:  1 
unit_conversion : Unit Class 7 (Volume/Time) cl_col:  CLFTAU1 CLFTAU2  parameters are scaled from  ML/HR  to  L/HR  via scaling factor:  0.001 
unit_conversion : Unit Class 8: (Amount.Time/Volume) auc_col:  AUCALL AUCLAST AUCLAST1 AUCLAST2 AUCLASTC1 AUCLASTC2 AUCINFO AUCINFOC AUCINFP AUCINFPC AUCTAU1 AUCTAU2  parameters are scaled from  UG.HR/ML  to  NG.HR/ML  via scaling factor:  1000 
unit_conversion : Unit Class 9: (Amount.Time.Time/Volume) aumc_col:  AUMCTAU1 AUMCTAU2  parameters are scaled from  UG.HR.HR/ML  to  NG.HR.HR/ML  via scaling factor:  1000 
unit_conversion : Unit Class 12: ([Amount/Volume]/Amount) concdn_col:  CMAXDN1 CMAXDN2  parameters are scaled from  UG/ML/MG  to  NG/ML/MG  via scaling factor:  1000 
There were 50 or more warnings (use warnings() to see the first 50)
> warnings()
Warning messages:
1: In run_M1_SS_computation(data = merged_data, map = map_data,  ... :
  Flag 'FLGEMESIS' is not present in the dataset
2: In unit_conversion(data = data_data, map = map_data, result = data_data,  ... :
  'CONCOUTPUTUNIT' is not present in the proper form! Please try again using 'Amount/Volume' format!
3: In unit_conversion(data = data_data, map = map_data, result = data_data,  ... :
  'CONCOUTPUTUNIT' is not present in the proper form! Please try again using 'Amount/Volume' format!
4: In unit_conversion(data = data_data, map = map_data, result = data_data,  ... :
  'CONCOUTPUTUNIT' is not present in the proper form! Please try again using 'Amount/Volume' format!
5: In unit_conversion(data = data_data, map = map_data, result = data_data,  ... :
  'CONCOUTPUTUNIT' is not present in the proper form! Please try again using 'Amount/Volume' format!
6: In unit_conversion(data = data_data, map = map_data, result = data_data,  ... :
  'CONCOUTPUTUNIT' is not present in the proper form! Please try again using 'Amount/Volume' format!
7: In unit_conversion(data = data_data, map = map_data, result = data_data,  ... :
  'CONCOUTPUTUNIT' is not present in the proper form! Please try again using 'Amount/Volume' format!
8: In unit_conversion(data = data_data, map = map_data, result = data_data,  ... :
  'CONCOUTPUTUNIT' is not present in the proper form! Please try again using 'Amount/Volume' format!
9: In unit_conversion(data = data_data, map = map_data, result = data_data,  ... :
  'CONCOUTPUTUNIT' is not present in the proper form! Please try again using 'Amount/Volume' format!
10: In unit_conversion(data = data_data, map = map_data, result = data_data,  ... :
  'CONCOUTPUTUNIT' is not present in the proper form! Please try again using 'Amount/Volume' format!
11: In unit_conversion(data = data_data, map = map_data, result = data_data,  ... :
```

**Comment by rudraya** [comment]
This issue was addressed in 02b2470, please let me know if it still occurs.

**Comment by jhhughes256** [comment]
### Update

This issue no longer occurs as of 02b2470.
